engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -162,7 +162,7 @@ private void processInput(EntityRef entity, CharacterMovementComponent character
         Quat4f viewRotation;
         switch (characterMovementComponent.mode) {
             case WALKING:
-                if (!playerCamera.isTracked()) {
+                if (!config.getRendering().isVrSupport()) {
```

Comment:
@indianajohn: so you are saying that the only way to get the renderingConfig injected in the field is to add it to the context in RegisterInputSystem? Otherwise it stays null? 

Bummer. I really don't like the coupling between rendering and input system. Conceptually they should really be separate. Let's leave things as they are then, so that there is less stuff to remove if one day we get to revisit this.